.github/tests/test_script.py

```diff
@@ -35,6 +35,41 @@ def load_recipe(path):
     return content
 
 
+def check_invalid_recipes(recipes, invalids, loaded, version):
```

Comment:
And then the test_script is just for the script type, and we can follow this test_<name> convention for upcoming schemas.

Reply:
exactly, every schema should have atleast one file valid and invalid Buildspec with list of examples to test all the valid and invalid cases. The method ``check_invalid_recipes`` and ``check_valid_recipes`` can work for any version provided the inputs are passed to this method. Hope this design makes sense.